### PR TITLE
nit(devtools): add info note on hydration of user data

### PR DESCRIPTION
**Pull request by dalechyn**
Closes https://github.com/wevm/frog/discussions/130.

### CODE REVIEW

File: ui/src/components/AuthDialog.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export function AuthDialog(props: AuthDialogProps) {
 
   return createPortal(
     <div
-      className="flex items-center justify-center p-6"
+      className="flex justify-center items-center p-6"
```
Comment (dalechyn) [inline ui/src/components/AuthDialog.tsx]:
```suggestion
      className="flex items-center justify-center p-6"
```

File: ui/src/components/AuthDialog.tsx [hunk 2]
```diff
@@ -54,13 +54,13 @@ export function AuthDialog(props: AuthDialogProps) {
       }}
     >
       <div
-        className="bg-background-100 relative flex flex-col gap-4 scrollbars p-6 border-gray-alpha-100 border"
+        className="flex relative flex-col gap-4 p-6 border bg-background-100 scrollbars border-gray-alpha-100"
```
Comment (dalechyn) [inline ui/src/components/AuthDialog.tsx]:
```suggestion
        className="bg-background-100 relative flex flex-col gap-4 scrollbars p-6 border-gray-alpha-100 border"
```

File: ui/src/components/AuthDialog.tsx [hunk 3]
```diff
@@ -54,13 +54,13 @@ export function AuthDialog(props: AuthDialogProps) {
...
         style={{ borderRadius: '1.5rem' }}
         ref={ref}
       >
         <button
           type="button"
-          className="bg-transparent text-gray-800 rounded-full flex items-center justify-center absolute hover:bg-gray-100 size-8"
+          className="flex absolute justify-center items-center text-gray-800 bg-transparent rounded-full hover:bg-gray-100 size-8"
```
Comment (dalechyn) [inline ui/src/components/AuthDialog.tsx]:
```suggestion
          className="bg-transparent text-gray-800 rounded-full flex items-center justify-center absolute hover:bg-gray-100 size-8"
```

File: ui/src/components/AuthDialog.tsx [hunk 4]
```diff
@@ -71,12 +71,12 @@ export function AuthDialog(props: AuthDialogProps) {
           <Cross1Icon />
         </button>
 
-        <h1 className="text-base font-bold text-gray-1000 text-center">
+        <h1 className="text-base font-bold text-center text-gray-1000">
```
Comment (dalechyn) [inline ui/src/components/AuthDialog.tsx]:
```suggestion
        <h1 className="text-base font-bold text-gray-1000 text-center">
```

File: ui/src/components/AuthDialog.tsx [hunk 5]
```diff
@@ -71,12 +71,12 @@ export function AuthDialog(props: AuthDialogProps) {
...
           Scan with Phone
         </h1>
 
         <p
-          className="text-sm text-gray-700 leading-snug text-center"
+          className="text-sm leading-snug text-center text-gray-700"
```
Comment (dalechyn) [inline ui/src/components/AuthDialog.tsx]:
```suggestion
          className="text-sm text-gray-700 leading-snug text-center"
```

File: ui/src/components/AuthDialog.tsx [hunk 6]
```diff
@@ -110,15 +110,15 @@ export function AuthDialog(props: AuthDialogProps) {
               role="alert"
               aria-live="assertive"
               aria-atomic="true"
-              className="flex items-center justify-center flex-col gap-4 absolute"
+              className="flex absolute flex-col gap-4 justify-center items-center"
```
Comment (dalechyn) [inline ui/src/components/AuthDialog.tsx]:
```suggestion
              className="flex items-center justify-center flex-col gap-4 absolute"
```

File: ui/src/components/AuthDialog.tsx [hunk 7]
```diff
@@ -110,15 +110,15 @@ export function AuthDialog(props: AuthDialogProps) {
...
               style={{
                 inset: '0',
               }}
             >
               <div className="font-medium text-gray-1000">Code timed out</div>
 
               <button
-                className="bg-gray-1000 border border-gray-200 py-2 px-4 text-gray-100 font-medium text-sm rounded-md"
+                className="py-2 px-4 text-sm font-medium text-gray-100 rounded-md border border-gray-200 bg-gray-1000"
```
Comment (dalechyn) [inline ui/src/components/AuthDialog.tsx]:
```suggestion
                className="bg-gray-1000 border border-gray-200 py-2 px-4 text-gray-100 font-medium text-sm rounded-md"
```

File: ui/src/components/AuthDialog.tsx [hunk 8]
```diff
@@ -130,13 +130,22 @@ export function AuthDialog(props: AuthDialogProps) {
 
         <button
           type="button"
-          className="bg-gray-100 border border-gray-200 p-3 text-gray-1000 font-medium text-sm rounded-xl mt-1"
+          className="p-3 mt-1 text-sm font-medium bg-gray-100 rounded-xl border border-gray-200 text-gray-1000"
```
Comment (dalechyn) [inline ui/src/components/AuthDialog.tsx]:
```suggestion
          className="bg-gray-100 border border-gray-200 p-3 text-gray-1000 font-medium text-sm rounded-xl mt-1"
```

**Review by dalechyn — COMMENTED** [review]
local formatter changes to undo.

**Comment by dalechyn** [comment]
Concerned code logic:
https://github.com/wevm/frog/blob/faae81ecd157b18fb2962c7a594d77ace13ca9eb/src/dev/devtools.tsx#L170-L185

**Comment by tmm** [comment]
Appreciate this! Let's put this in the docs instead so it keeps the interface clean.

**Pull request closed without merging**